internal/cli/artifact_build.go

```diff
@@ -41,6 +42,46 @@ func (c *ArtifactBuildCommand) Run(args []string) int {
 			app.UI.Output("\nCreated artifact v%d", buildResult.Push.Sequence)
 		}
 
+		// Show input variable values used in build
+		app.UI.Output("Variables used:", terminal.WithHeaderStyle())
+		headers := []string{
+			"Variable", "Value", "Type", "Source",
+		}
+
+		tbl := terminal.NewTable(headers...)
+		// sort alphabetically for joy
+		inputVars := make([]string, 0, len(buildResult.Build.VariableRefs))
+		for iv := range buildResult.Build.VariableRefs {
+			inputVars = append(inputVars, iv)
+		}
+		sort.Strings(inputVars)
+		for _, iv := range inputVars {
+			// We add a line break in the value here because the Table word wrap
+			// alone can't accomodate the column headers to a long value
+			val := buildResult.Build.VariableRefs[iv].Value
+			if len(val) > 45 {
+				for i := range val {
+					// line break every 45 characters
+					if i%46 == 0 && i != 0 {
+						val = val[:i] + "\n" + val[i:]
```

Comment:
As is, the first chunk would be 46 chars because of indexing.

```suggestion
						val = val[:i-1] + "\n" + val[i:]
```

Reply:
I'm just going to make this `i%45`; the indexing 😵 got me but that's what I want -- up to the first 45 chars inclusive (index 0-44), and then the next set exclusive ([45:].

Reply:
Ah, I tested this and if you do i%45 it doesn't crop right! The first chunk ends up as 45 chars but every subsequent chunk is only 44. @krantzinator